mc2/controllers/base/templates/controller_edit.html

```diff
@@ -3,17 +3,33 @@
 
 {% block script %}
     <script type="text/javascript">
+      function guid() {
+        function s4() {
+          return Math.floor((1 + Math.random()) * 0x10000)
+            .toString(16)
+            .substring(1);
+        }
+        return s4() + s4() + '-' + s4() + '-' + s4() + '-' +
+          s4() + '-' + s4() + s4() + s4();
+      }
```

Comment:
This isn't technically a valid UUID (see https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_.28random.29 for the bits that specify variant and version), but that doesn't really matter much for our purposes unless Django actually validates that.